extensions/amp-nested-menu/0.1/amp-nested-menu.js

```diff
@@ -274,6 +274,8 @@ export class AmpNestedMenu extends AMP.BaseElement {
         break;
       case Keys.UP_ARROW: /* fallthrough */
       case Keys.DOWN_ARROW:
+      case Keys.HOME:
+      case Keys.END:
         this.handleVerticalArrowKeyDown_(e);
```

Comment:
I recommend renaming `handleVerticalArrowKeyDown_` since it is no longer handling just arrow keys.